### PR TITLE
added login page

### PR DESCRIPTION
**Pull request by Priyanshi-Tiwari**
Stellar_Login_Page

### CODE REVIEW

**Review by shivangisharma01 — APPROVED** [review]
The code is clean and understandable, looks good to merge.

**Review by RahulSingh9131 — APPROVED** [review]
everything looks good to me. Go ahead and merge.

File: pages/loginpage.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+              <li class="list-inline"><a class="links" target="_blank" href="https://twitter.com/Priyanshicodes">Shipping & Returns</a></li>
+              <li class="list-inline"><a class="links" target="_blank"  href="https://github.com/Priyanshi-Tiwari">Contact us</a></li>
+          </div>
+          <ul class="list-non-bullet">
+            <li class="list-inline">
+              <a
+                class="links"
+                target="_blank"
+                href="https://github.com/Priyanshi-Tiwari"
+                ><i class="fa-brands fa-github" style="font-size: 40px"></i
```
Comment (dev-enforced) [inline pages/loginpage.html:99]: Avoid applying inline styles you can use a class for it and use it here in the HTML

**Review by dev-enforced — APPROVED** [review]
The code looks good to me !! You can merge it !! Good work on the login component

**Pull request merged**